src/page/template/content/conversation/message-list.htm

```diff
@@ -1,13 +1,13 @@
  <div id="message-list" class="message-list" data-bind="with: $root.messageList, fadingscrollbar, event: {mousewheel: $root.messageList.onMouseWheel, scroll: $root.messageList.onScroll}">
   <div class="messages"
         data-bind="css: {'flex-center': verticallyCenterMessage},
-                   foreach: {data: conversation().messages_visible, as: 'message'}">
+                   foreach: {data: conversation().messages_visible, as: 'message', noChildContext: true}">
```

Comment:
As of knockout 3.5.0 we can now use the foreach without creating a new scope (and thus accessing the viewmodel's methods without relying on the `$parent` attribute).
https://github.com/knockout/knockout/pull/907#issuecomment-346955622